src/build/__main__.py

```diff
@@ -47,7 +47,7 @@ def _init_colors() -> Dict[str, str]:
 _STYLES = _init_colors()
 
 
-def _cprint(fmt='', msg=''):
+def _cprint(fmt: Optional[str] = '', msg: Optional[str] = ''):
```

Comment:
This isn't optional, it would type error if None.

Reply:
Just copy/pasted from below. After reading https://docs.python.org/3/library/typing.html#typing.Optional about how to annotate default params I guess it is fixed.

Reply:
Yeah, Optional really means `str | None` and isn’t tried to optional parameters (unless they are defaulted to None, for example).  Some people who shall not be names hate the name Optional for it. ;)